pkg/db/migration/000009_model_trigger.up.sql

```diff
@@ -0,0 +1,40 @@
+BEGIN;
+
+create type valid_trigger_status as enum ('TRIGGER_STATUS_COMPLETED', 'TRIGGER_STATUS_FAILED', 'TRIGGER_STATUS_PROCESSING', 'TRIGGER_STATUS_QUEUED');
+create type valid_trigger_source as enum ('TRIGGER_SOURCE_CONSOLE', 'TRIGGER_SOURCE_API');
+
+create table model_trigger
+(
+    uid                 uuid primary key,
+    model_uid           uuid                                                                    not null,
+    trigger_uid         uuid                                                                    not null,
+    model_version       varchar(255)                                                            not null,
+    model_task          valid_task                                                              not null,
+    model_tags          jsonb                                                                   not null,
+    status              valid_trigger_status                                                    not null,
+    visibility          valid_visibility         default 'VISIBILITY_PRIVATE'::valid_visibility not null,
+    source              valid_trigger_source                                                    not null,
+    start_time          timestamp with time zone default CURRENT_TIMESTAMP                      not null,
+    total_duration      bigint,
```

Comment:
I've investigated postgresql `interval` type but it will require more extra code(or an extra module import) to work with gorm than int.
There is a comment below for marking the unit.